pacmacs.el

```diff
@@ -130,18 +131,58 @@
         :column column
         :type 'wall))
 
-(defun pacmacs--make-pill (row column anim-name points)
+(defun pacmacs--make-pill (row column anim-name points type)
```

Comment:
Hm, I see, the general `make-object` function should take a lot more parameters. Well, then you could ignore the previous comment.